brave-http-common/src/test/java/smartthings/brave/propagation/SimpleB3ContextCarrierEncodingTest.java

```diff
@@ -135,7 +137,7 @@ public void injectingFlagsShouldNotUnsetBuilderValues() {
     SimpleB3ContextCarrier.Setter setter = new SimpleB3ContextCarrier.Setter();
     setter.put(carrier, SimpleB3ContextCarrier.TRACE_ID_NAME, HexCodec.toLowerHex(1234));
     setter.put(carrier, SimpleB3ContextCarrier.SPAN_ID_NAME, HexCodec.toLowerHex(9012));
-    setter.put(carrier, SimpleB3ContextCarrier.FLAGS_NAME, "0000000000000000");
+    setter.put(carrier, SimpleB3ContextCarrier.FLAGS_NAME, "0000000000000000"); // TODO 0 is not a valid hex anymore
```

Comment:
is it zero is not valid? or an unnecessarily long string of zeros :)

Reply:
The brave `HexCodec` class throws a `NumberFormatException` if you try to encode hex that ends up equaling 0